browser/src/control/Control.TopToolbar.js

```diff
@@ -275,6 +278,11 @@ L.Control.TopToolbar = L.Control.extend({
 				window.hideTooltip(this, e.target);
 			},
 			onRefresh: function(event) {
+				if (event.item && event.item.invisible == true) {
+					var toolItem = $(this.box).find('#tb_'+ this.name +'_item_'+ w2utils.escapeId(event.item.id));
+					toolItem.css('visibility', 'hidden');
+				}
+
 				if ((event.target === 'styles' || event.target === 'fonts' || event.target === 'fontsizes') && event.item) {
 					var toolItem = $(this.box).find('#tb_'+ this.name +'_item_'+ w2utils.escapeId(event.item.id));
```

Comment:
Does it run without errors? I think there is `var toolItem` redeclared. `var` should be used only in the first occurence in the function according to our checkers.

Reply:
also we could have only one place with getting this item (now we duplicated this line of code).
Like:
```
if (event.item)
    var toolItem = $(this.box).find('#tb_'+ this.name +'_item_'+ w2utils.escapeId(event.item.id));
 if (toolItem && event.item.invisible == true)
    ....
 if (toolItem && (event.target === 'styles' || event.target === 'fonts' || event.target === 'fontsizes'))
    ....
 ```